docs/Reference/API-Objects.md

```diff
@@ -34,6 +34,17 @@ Returned by [`eth_getBlockByHash`](API-Methods.md#eth_getblockbyhash) and
 | **uncles**           | Array                 | Array of uncle hashes.                                             |
 | **baseFeePerGas**    | Quantity              | The block's [base fee per gas](../Concepts/Transactions/Transaction-Types.md#eip1559-transactions). This field is empty for blocks created before [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md). |
 
+## Fee history results object
+
+Returned by [`eth_feeHistory`](API-Methods.md#eth_feehistory) for the requested block range.
+If not all blocks are available, only the available subsection of the requested range will be returned.
+
+| Key | Type | Value |
+|-----|:----:|-------|
+| **oldestBlock**        | Quantity, Integer     | Lowest number block of the returned range.                        |
+| **baseFeePerGas**      | Array                 | Array of block base fees per gas. This includes the next block after the newest of the returned range, because this value can be derived from the newest block. Returns zeroes for blocks created before [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md). |
```

Comment:
This reads a bit awkwardly. I understand what it means, but needed to ready through multiple times